server/src/models/Nest.js

```diff
@@ -164,7 +164,8 @@ class Nest extends Model {
    * @param {ReturnType<typeof import("server/src/utils/getBbox").getBboxFromCenter>} bbox
    * @returns {Promise<FullNest[]>}
    */
-  static async search(perms, args, distance, bbox) {
+  // eslint-disable-next-line no-empty-pattern
+  static async search(perms, args, {}, distance, bbox) {
```

Comment:
Why add in a dummy parameter? Why not just remove it from the calling sites?

Reply:
Dosn't removing it from calling sites brakes this search?
https://github.com/WatWowMap/ReactMap/commit/adf09fcedf373127070de9e13264e0dbd4d08be3#r160294066

Reply:
that's the point to my question  - if the parameter is never used why is it even provided?

Reply:
Of course there may be some complex something or other that requires a whole lot of functions to have the same signature, all called by search - but I thought it was worth asking the question because it's a very odd thing to do :-)

Reply:
Why no fix? :(

Reply:
I guess this was used here in `DbManager`:

```js
        this.models[model].map(async ({ SubModel, ...source }) =>
          SubModel[method](
            perms,
            args,
            source,
            DbManager.getDistance(args),
            bbox,
          ),
        ),
```

Looks like this has always been here since 2022?